.github/workflows/release_snapshot.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
```

Comment:
Should we also update this to publish snapshots off of `main` instead of `master`?